src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -62,7 +63,7 @@ const BUTTON_CLASSNAME = b('actions-button');
 export function withTableActions<I extends TableDataItem, E extends {} = {}>(
     TableComponent: React.ComponentType<TableProps<I> & E>,
 ): React.ComponentType<TableProps<I> & WithTableActionsProps<I> & E> {
-    const componentName = TableComponent.displayName || TableComponent.name || 'Component';
+    const componentName = getComponentName(TableComponent);
```

Comment:
Please extract everything related to `getComponentName` into another PR.

Reply:
Done #257